### DISCUSSION TITLE
Graphics card requirements

### DISCUSSION THREAD

**Discussion by jhonayala** [prasanthrangan/hyprdots#1461] [Q&A] [answered]
hello.  I would like to know what the minimum graphics card requirements are.

**Comment by ghost** [comment, marked as answer]
Hyprland wants OpenGL v3.0+ for the GPU for everything to render properly, if your GPU is lower then that, you'll want to use `hyprland-legacyrenderer`, which is maintained in the AUR by @kRHYME7 and me. That will however cause some graphical effects not to work properly, so there isn't really a minimum technically but it's preferred you meet at least OpenGL v3.0+
